### PR TITLE
ISSD-2993 [PRM] Add arrow down from Clay Panel in Activity Card

### PR DESCRIPTION
**Pull request by abfranca**

### CODE REVIEW

**Comment by abfranca** [comment]
ci:test:sf

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/mdf-request-detail-collection/fragments/activities-list-panel/index.css
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/mdf-request-detail-collection/fragments/activities-list-panel/index.css:1]: Hey @abfranca. Is it possible to use without define a css style? I mean, just using the classes provided by Clay.

**Review by jairmedeiros — CHANGES_REQUESTED** [review]
Lets try to remove the custom css styling.

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/mdf-request-detail-collection/fragments/activities-list-panel/index.css
Comment (abfranca) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/mdf-request-detail-collection/fragments/activities-list-panel/index.css:1]: Done!

**Comment by abfranca** [comment]
ci:test:sf

**Comment by abfranca** [comment]
ci:test:sf

**Comment by jairmedeiros** [comment]
ci:forward:force

**Pull request closed without merging**